### PR TITLE
Update rust to 2018 edition

### PR DESCRIPTION
**Pull request by povilasb**
Makes builds simpler cause we don't have to track the Rust versions netsim builds on. Now it's supposed to build on all latest stable Rust  2018 edition versions.

### CODE REVIEW

File: src/pcap.rs [hunk 1]
```diff
@@ -29,7 +29,7 @@ impl IpLog {
         const VERSION_MINOR: u16 = 4;
         const LINKTYPE_RAW: u32 = 101;
 
-        let try = || -> io::Result<_> {
+        let r#try = || -> io::Result<_> {
```
Comment (canndrew) [inline src/pcap.rs]: This should probably just be called something else.

**Comment by canndrew** [comment]
I left one small nit on this PR, but it otherwise looks good.

File: src/pcap.rs (hunk 1 above)
Comment (povilasb) [reply src/pcap.rs]: You right, didn't notice, will change :)

**Comment by povilasb** [comment]
Pushed the changes ;)

**Pull request merged**